src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/IsNullOrEmptyConverter.shared.cs

```diff
@@ -18,10 +24,25 @@ public class IsNullOrEmptyConverter : ValueConverterExtension, IValueConverter
 		/// <param name="parameter">Additional parameter for the converter to handle. This is not implemented.</param>
 		/// <param name="culture">The culture to use in the converter. This is not implemented.</param>
 		/// <returns>A <see cref="bool"/> indicating if the incoming value is null or empty.</returns>
-		public object Convert(object? value, Type? targetType, object? parameter, CultureInfo? culture) => ConvertInternal(value);
+		public object Convert(object? value, Type? targetType, object? parameter, CultureInfo? culture)
+			=> InvertCheck != ConvertInternal(value);
+
+		internal static bool ConvertInternal(object? value, bool isListCheck = false)
+		{
+			if (value == null)
+				return true;
+
+			if (value is IEnumerable list)
```

Comment:
String is an array of `chars`, so this `if` will be true for all strings. 
1) This means that  `value is string str` bellow is redundant
2) This method will return `false` for string that contains whitespace, while previously it returned `true` in such cases

Reply:
Done!